hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -259,10 +259,14 @@
           desc: "Set to 1 if an integrity or consistency check times out. This raises an otp_check_failed alert and is an unrecoverable error condition."
         }
         { bits: "10"
+          name: "AUX_FSM_ERROR"
+          desc: "Set to 1 if either of the following FSMs has reached an invalid state: LFSR timer, scrambling datapath, key derivation module. This raises an otp_check_failed alert and is an unrecoverable error condition."
```

Comment:
there might not be a need for this, but is there any way to tell which of the FSMs reached an invalid state?

Reply:
I initially had this split out into individual bits, but then thought this might be overkill. the information is there, though, so we could just as well expose it.

Reply:
i don't really think there's a need for it... because it feels like if any of these go wrong, the system is toast.

Reply:
the errors are split out as individual bits now.